cmd/agent/app/server.go

```diff
@@ -81,9 +81,10 @@ func InitializeTLS(s *options.AgentOptions) (*agent.TLSOptions, error) {
 }
 
 // ServeHealthProbes starts a server to check healthz and readyz probes
-func ServeHealthProbes(stop <-chan struct{}, healthProbeBindAddress string) {
+func ServeHealthProbes(stop <-chan struct{}, healthProbeBindAddress string, configCheck healthz.Checker) {
 	healthzHandler := &healthz.Handler{Checks: map[string]healthz.Checker{
 		"healthz-ping": healthz.Ping,
+		"configz-ping": configCheck,
```

Comment:
should we add configz-ping into livenessProbe of pods? 
if yes, need to update work-manger chart https://github.com/open-cluster-management/klusterlet-addon-operator/blob/56609f79aed359696db77ae4abea67e2449e690c/helm-charts/work-manager/templates/deployment.yaml#L81.  and for e2e tests